docs/pages/admin-guides/management/guides/aws-iam-identity-center/using-aws-cli.mdx

```diff
@@ -0,0 +1,209 @@
+description: Provides an overview of the Teleport AWS IAM Identity Center integration.
+labels:
+ - how-to
+ - identity-governance
+---
+
+This guide will show you how to configure the `aws` command-line tool to use
+access granted via Teleport and AWS Identity Center.
+
+## How it works
+
+For a deep dive into how Teleport manages AWS Identity Center access works you can
+read the main [AWS IAM Identity Center guide](./aws-iam-identity-center.mdx). For the 
+purposes of this guide, it's enough to understand that Teleport manages the creation
+and deletion of AWS Account Assignments based on a users's Account Assignment grants,
+either from their standing Teleport Roles, Access List membership or approved
+Access Requests. 
+
+You can access these Teleport-managed Accounts and Permission Set assignments
+with the AWS CLI tools by using `sso` login and AWS profiles.
+
+## Prerequisites
+
+Before you begin, you will need:
+
+ * A Teleport-managed AWS Identity Center organization. See our [getting started guide](./guide.mdx)
+   for setting up an Identity Center integration.
+ * The AWS CLI tools, installed as per the AWS [installation guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+ * The SSO Start URL and AWS Region for your Identity Center organization. Ask your AWS administrator for the appropriate values.
+
+## Step 1/2. Configure the AWS SSO Session
+
+This step configures the AWS CLI tools to use SSO for authentication. You can
+either configure this manually, or via the `aws configure sso-session` wizard.
+
+<Tabs>
+<TabItem label="Wizard">
+Invoke the AWS SSO configuration wizard by running the following command and
+answering the questions asked by the wizard.  
+
+```shell
+$ aws configure sso-session
+```
+
+You will need to pick a name for the SSO session; this is just a local name for
+this particular SSO configuration. For this example we are using `my-identity-center`. 
+
+```shell
+$ aws configure sso-session
+SSO session name: my-identity-center
+SSO start URL [None]: https://d-12234567890.awsapps.com/start
+SSO region [None]: us-east-1
+SSO registration scopes [sso:account:access]:
+```
+</TabItem>
+<TabItem label="Manual Configuration">
+You can configure SSO authentication manually by editing your `.aws/config` file. The above example would look like this:
+
+```ini
+[sso-session my-identity-center]
+sso_start_url = https://d-12234567890.awsapps.com/start
+sso_region = us-east-1
+sso_registration_scopes = sso:account:access
+```
+</TabItem>
+</Tabs>
+
+### Testing the SSO Session
+
+To test the SSO configuration, try logging into AWS via SSO.
+
+```shell
+$ aws sso login --sso-session my-identity-center
+```
+
+This will launch a browser-based flow that will log you into AWS via Teleport
+and ask you confirm the AWS CLI tool's access to AWS using your account.
+
+## Step 2/2. Creating AWS Profiles
+
+You will need to create a separate AWS profile for each AWS Account and Permission Set
```

Comment:
Nit: To avoid confusion with something you need to create in AWS management console.
```suggestion
You will need to create a separate AWS CLI profile for each AWS Account and Permission Set
```